plugins/katello/3.16/user_guide/remote_databases/index.md

```diff
@@ -26,6 +26,15 @@ There are two ways to deploy Foreman with remote databases:
 
 In either scenario, both of the databases don't have to be remote. You can opt to use only a remote mongo database or only a remote postgresql database. Both postgresql and mongo databases can be on the same host, but this isn't recommended due to the amount of resources mongo can use.
 
+
+### Important notes:
+
+1. Pulpcore installs a separate PostgreSQL database and lacks support for external databases. Databases should be upgraded to avoid conflicts.
```

Comment:
Version 3.16 does support external databases
```suggestion
```

Reply:
This was marked as resolved, but 3.16 does support a separate PG DB for Pulpcore. If you keep a list, I'd suggest not using numbers but rather bullets.